tests/baselines/reference/enumConstantMemberWithTemplateLiterals.errors.txt

```diff
@@ -50,4 +50,10 @@ enumConstantMemberWithTemplateLiterals.ts(28,15): error TS2363: The right-hand s
         b = `1` + `1`,
         c = "2" + `1`
     }
+    
+    declare enum T8 {
+      X = `${1}`,
+      Y = `${true}`,
+      Z = `pre${!false}middle${"foo" + true}`,

```

Comment:
If the PR gets softly accepted - I should add more test cases here. I didn't want to spend time on it yet in case it gets rejected